lib/api/controllers/documentController.js

```diff
@@ -194,14 +194,14 @@ class DocumentController extends NativeController {
       format,
       fields);
 
-    let mimeType = 'application/json';
+    let mimeType = 'html/text';
 
     switch (format) {
       case 'csv':
         mimeType = 'text/csv';
         break;
       default:
-        mimeType = 'application/json';
+        mimeType = 'html/text';
```

Comment:
Why this type?

Reply:
Because JSONL is not really a JSON file and could not be parsed like so, if it's defined as `application/json` browser and other HTTP libraries might want to parse it as JSON and will fail